README.md

```diff
@@ -52,6 +53,11 @@ Search links that point directly to suitable issues to contribute to on GitHub
 - [GitHub Learning Resources](https://help.github.com/articles/git-and-github-learning-resources/) - Git and GitHub learning resources
 - [Pro Git](https://git-scm.com/book/en/v2) - The entire Pro Git book, written by Scott Chacon and Ben Straub and published by Apress.
 - [Try Git](https://try.github.io/) - Learn Git in your browser for free
+- [Udacity](https://www.udacity.com/course/how-to-use-git-and-github--ud775) - Free git online course
```

Comment:
Change the alt text from just "Udacity" to "How to Use Git and GitHub". Additionally, change "Free git online course" to be more descriptive of what how this course is different than the others. Like what you expect to do in the course to learn git.